pandas/core/internals.py

```diff
@@ -1690,6 +1690,17 @@ def _nanpercentile(values, q, axis, **kw):
                               placement=np.arange(len(result)),
                               ndim=ndim)
 
+    def _replace_coerce(self, mask=None, dst=None, convert=False):
+        """replace value to dst where mask is true, value is coerce to target
```

Comment:
can you add a full Parameters section here

Reply:
Do you mean add **kwargs and *args ?

Reply:
@jreback Or do you mean add a full docsstring with all parameters ?

Reply:
yes full doc string